models/src/node/crd/mod.rs

```diff
@@ -26,19 +26,39 @@
 /// };
 /// ```
 ///
+/// Add the BottlerocketShadow version to the end of BOTTLEROCKETSHADOW_CRD_VERSIONS
+/// So the webhook conversion could set up proper conversion_review_versions
+#[cfg_attr(doctest, doc = " ````no_test")]
+/// ```
+/// static ref BOTTLEROCKETSHADOW_CRD_VERSIONS: Vec<String> = {
```

Comment:
Should remove this?

Reply:
Sorry I was confused what do you want me to remove?

Reply:
Oh, I saw you commented out few lines. and I was wondering if you don't need them? 
like
```
/// static ref BOTTLEROCKETSHADOW_CRD_VERSIONS: Vec<String> = {
....
```

Reply:
This is the doc for the file.